packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js

```diff
@@ -0,0 +1,6 @@
+export const STOP_INSTANCE_INFO_LINK =
```

Comment:
**nitpick(blocking):** We cannot add the link only in French. This has to be added in all languages.

Reply:
I'll check this with the PO.

Reply:
@anooparveti 
The links to the other subsidiaries is addded in the last commit.